packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,33 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## October
 
+### Weekly release 2022.10.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Invalid phone numbers rejected](#invalid-phone-numbers-rejected)                | October 26, 2022         |
+| [Bugs fixed in 2022.10.2](#bugs-fixed-in-2022-10-2)                      | October 26, 2022         |
+
+#### Invalid phone numbers rejected
+
+Okta now rejects attempts to enroll a toll-free, premium, fixed-line (SMS), or any other invalid or unrecognized phone number. This ensures that only valid phone numbers are used for multifactor authentication or device enrollment. See [Configure and use telephony](https://help.okta.com/okta_help.htm?type=oie&id=ext-telephony-how-to) or the [Enroll Factor API](/docs/reference/api/authn/#enroll-factor). <!--OKTA-437305-->
+
+#### Bugs fixed in 2022.10.2
+
+* A `/token` request that used the `code` parameter failed when the **Groups claim type** in the app was set to **Expression** and the group number exceeded 100. (OKTA-518536)
+
+* A `/token` request that used the `code` parameter and the `refresh_token` grant type failed when the **Groups claim type** in the app was set to either **Expression** or **Filter** and the group number exceeded 100. (OKTA-531605)
+
+* When a user entered their credentials in the widget (version 6.6.1) during self-service registration, their information was cleared if the username didn't exist in the org. (OKTA-532293)
+
+* The MyAccount API sent no notifications for Add Email calls with a false `sendEmail` parameter. (OKTA-539268)
+
+* A user with a current session in Classic Engine couldn't be deleted (`DELETE /session/me`) after upgrading to Identity Engine. (OKTA-517093)
```

Comment:
```suggestion
* An opened session in Classic Engine couldn't be deleted (`DELETE /session/me`) after upgrading to Identity Engine. (OKTA-517093)
```
I know this was already approved, so you can ignore this suggestion. The original sentence sounds like you are trying to delete the user (not the session). I interpret "with a current session in Classic Engine" as a modifier.

Reply:
Great catch. I wanted to change this one, too. I'll pass by Graham for another look for all of these.